### PR TITLE
Feature#4 form validations with yup

### PR DESCRIPTION
**Pull request by hancel-avila**
# Description
 - We added Yup to build a LogIn and Signup schema for validation.
 - We used Formik to handle both validations that prints error messages in case a current input field is empty.
 - This will be helpful for future handle submissions and also a visual assistance for users when they missed to fill an input.

### CODE REVIEW

File: src/constants/schemas/loginSchema.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/* eslint-disable prettier/prettier */
```
Comment (cfcortes) [inline src/constants/schemas/loginSchema.js:1]: Codacy has a fix for the issue: ['prettier/prettier' rule is disabled but never reported.](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7886201)
```suggestion

```

File: src/constants/schemas/signupSchema.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable prettier/prettier */
```
Comment (cfcortes) [inline src/constants/schemas/signupSchema.js:1]: Codacy has a fix for the issue: ['prettier/prettier' rule is disabled but never reported.](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7886201)
```suggestion

```

File: src/components/TextInput/index.js [hunk 3]
```diff
@@ -1,15 +1,40 @@
...
+  const {
+    label,
+    handleChange,
+    handleBlur,
+    values,
+    formControlName,
+    errors,
+    touched,
+  } = props;
+  console.log(label);
```
Comment (YilKanda) [inline src/components/TextInput/index.js:20]: Delete this console.log()

File: src/components/TextInput/index.js [hunk 4]
```diff
@@ -1,15 +1,40 @@
...
 
-const TextInputField = ({label}) => {
-    return (
-        <View>
-            <Text>{label}</Text>
-            <TextInputCustom/>
-        </View>
-    );
+const TextInputField = props => {
+  console.log(props);
```
Comment (YilKanda) [inline src/components/TextInput/index.js:10]: Delete this console.log()

File: src/containers/SignUp/index.js [hunk 5]
```diff
@@ -1,31 +1,58 @@
...
+      <Formik
+        validationSchema={signUpValidationSchema}
+        initialValues={{
+          name: '',
+          email: '',
+          password: '',
+          agreeTerms: false,
+        }}
+        validateOnMount={true}
+        onSubmit={values => console.log(values)}>
```
Comment (YilKanda) [inline src/containers/SignUp/index.js:25]: If the console.log() is not necessary, delete it.

File: src/containers/Login/index.js [hunk 6]
```diff
@@ -1,24 +1,45 @@
...
+      <Formik
+        validationSchema={logInValidationSchema}
+        initialValues={{
+          name: '',
+          email: '',
+          password: '',
+          agreeTerms: false,
+        }}
+        validateOnMount={true}
+        onSubmit={values => console.log(values)}>
```
Comment (YilKanda) [inline src/containers/Login/index.js:20]: If the console.log() is not necessary, delete it.

**Review by YilKanda — CHANGES_REQUESTED** [review]
When uploading your code to GitHub, remove the console.log(), as they should only function to test that the code is working while it is being done, but not leave them in the master branch.

**Review by DemisRincon — APPROVED** [review]

**Review by yovanha — APPROVED** [review]

**Review by cfcortes — APPROVED** [review]

**Pull request merged**